.travis.yml

```diff
@@ -15,5 +15,5 @@ before_install:
   - tools/travis/nuke_pg.sh
 install:
   - tools/travis/install_pg.sh
-script: tools/travis/pg_travis_multi_test.sh
+script: CFLAGS=-Werror tools/travis/pg_travis_multi_test.sh
```

Comment:
On 2016-08-03 13:57:20 -0700, Marco Slot wrote:
> > @@ -15,5 +15,5 @@ before_install:
> >    - tools/travis/nuke_pg.sh
> >  install:
> >    - tools/travis/install_pg.sh
> > -script: tools/travis/pg_travis_multi_test.sh
> > +script: CFLAGS=-Werror tools/travis/pg_travis_multi_test.sh
> 
> Would it make sense to put this into the script or are there cases where we don't want -Werror? It looks a bit out-of-place here.

Setting -Werror is a bad idea imo - it varies *HEAVILY* between compiler
versions, flags, and optimization settings.